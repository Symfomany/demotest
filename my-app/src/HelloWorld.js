import React, { Component } from "react";

class HelloWorld extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  render() {
    return <p>Hello World!</p>;
  }
}

export default HelloWorld;
