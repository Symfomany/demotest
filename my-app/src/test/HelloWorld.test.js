import HelloWorld from "../HelloWorld";
import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe(HelloWorld, () => {
  it("renders and matches our snapshot", () => {
    const component = renderer.create(<HelloWorld name="Person" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // component.find('button.frenchify').simulate('click');
    // component.find('button.remove').simulate('click');
    // expect(mockRemoveGreeting).toBeCalled();
    //expect(component.text()).toContain(name);
  });
});
