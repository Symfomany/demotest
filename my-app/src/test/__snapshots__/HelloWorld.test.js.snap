// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class HelloWorld extends _react.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  render() {
    return _react2.default.createElement("p", { __source: { fileName: _jsxFileName, lineNumber: 9 }, __self: this }, "Hello World!");
  }} renders and matches our snapshot 1`] = `
<p>
  Hello World!
</p>
`;
